utils/CoinbaseHelper.js

```diff
@@ -59,6 +59,7 @@ class CoinbaseHelper {
             };
             authedClient.placeOrder(buyParams).then(r => {
                 console.log(r);
+		        //Telegram.init().telegramPost(r);
```

Comment:
Make a variable in Settings.js named isTelegramNotificationOn
Check with an if statement and remove the comment